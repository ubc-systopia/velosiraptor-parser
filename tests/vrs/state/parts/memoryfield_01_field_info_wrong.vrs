mem foo [ 0 ] {Layout {}}