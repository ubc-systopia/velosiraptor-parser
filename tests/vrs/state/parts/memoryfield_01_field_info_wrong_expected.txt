error: expected `identifier`, found `0`
     --> tests/vrs/state/parts/memoryfield_01_field_info_wrong.vrs:1:11
      |
    1 |  mem foo [ 0 ] {Layout {}}
      |            ^ help: add `identifier` here.
